doc/manual/source/man/dnst-notify.rst

```diff
@@ -0,0 +1,50 @@
+dnst-notify
+===============
+
+Synopsis
+--------
+
+:program:`dnst notify` [``OPTIONS``] ``-z <ZONE>`` ``<SERVERS>...``
+
+Description
+-----------
+
+**dnst notify** sends a NOTIFY message to the specified name servers. A name
+server can be specified as a domain name or IP address.
+
+This tells them that an updated zone is available at the primaries. It can
+perform TSIG signatures, and it can add a SOA serial number of the updated
+zone. If a server already has that serial number it will disregard the message.
+
+Options
+-------
+
+.. option:: -z <ZONE>
+
+      The zone to send the NOTIFY for.
+
+.. option:: -s <SOA VERSION>
+
+      SOA version number to include in the NOTIFY message.
+
+.. option:: -y, --tsig <name:key[:algo]>
```

Comment:
Should `name`, `key`, `algo` be uppercase?

Reply:
For consistency, probably yes.